Library/Homebrew/utils/github.rb

```diff
@@ -70,15 +74,22 @@ def initialize(github_message)
           The GitHub credentials in the macOS keychain may be invalid.
           Clear them with:
             printf "protocol=https\\nhost=github.com\\n" | git credential-osxkeychain erase
-          Or create a personal access token:
-            #{ALL_SCOPES_URL}
-          #{Utils::Shell.set_variable_in_profile("HOMEBREW_GITHUB_API_TOKEN", "your_token_here")}
+          #{CREATE_GITHUB_PAT_MESSAGE}
         EOS
       end
       super message.freeze
     end
   end
 
+  # Error when the user has no GitHub API credentials set at all (macOS keychain or envvar).
+  class MissingAuthenticationError < Error
+    def initialize
+      message = +"No GitHub credentials found in Keychain or environment.\n"
```

Comment:
```suggestion
      message = +"No GitHub credentials found in macOS Keychain or environment.\n"
```

although perhaps the message could be customised per OS?

Reply:
Let's do this for now - if people who use Linux more regularly than I do these days want to customise this, they can. 🙏🏻